src/request.rs

```diff
@@ -0,0 +1,23 @@
+use http::Request;
+use reqwest::Response;
+use std::convert::TryFrom;
+
+pub struct InfinispanRequest {
+    request: Request<String>,
+    http_client: reqwest::Client,
+}
+
+impl<'r, 'c> InfinispanRequest {
+    pub fn new(request: Request<String>, http_client: reqwest::Client) -> InfinispanRequest {
+        InfinispanRequest {
+            request,
+            http_client,
+        }
+    }
+
+    pub async fn send(self) -> Result<Response, reqwest::Error> {
+        self.http_client
+            .execute(reqwest::Request::try_from(self.request).unwrap())
```

Comment:
Avoid that unwrap() with a ? or map_err or chain_err or similar?
I use `#![warn(clippy::unwrap_used)]` (in crate bin/lib root file) to keep myself honest on these.

Reply:
Yes, this is a first version only, so I didn't worry too much about errors yet. But we'll definitely need to use something like `thiserror` (like we are doing in Limitador).